### PR TITLE
Changes told on 07-12

### PR DESCRIPTION
**Pull request by akhilvenugopal**
c

### CODE REVIEW

**Comment by akhilvenugopal** [comment]
c

File: connections.py [hunk 1]
```diff
@@ -25,12 +29,20 @@ def delete_G(servern, group):
...
     serv = str(serv)
     c.execute("SELECT groupn, tags, update1 FROM table1 where server = ?", (serv,))
     data = c.fetchall()
     return data
+
+def delete_tag(current_server,new_tags,group_name):
+    group_name = str(group_name)
+    new_tags=str(new_tags)
+    current_server= str(current_server)
+    c.execute('''UPDATE table1 SET tags = ? WHERE groupn = ? AND server = ?''', [new_tags, group_name,current_server,])
```
Comment (ajmalmsali) [inline connections.py]: why '''?

File: Main.py [hunk 2]
```diff
@@ -161,11 +163,44 @@ def delete_group(self):
...
+            temp = connections.group()                              #index of the group which tag is
+            cell = self.tableWidget.item(selected_row,0).text()     #tag name from table
+            indx = tags[temp].index(cell)                           #index of tag in group[num]
+            current_server = connections.server()
+            group_name = group[temp]
+            new_tags = tags[temp]
+            QMessageBox.about(self, "Remove", "Remove the tag ?")
+            del new_tags[indx]
+            connections.delete_tag(current_server,new_tags,group_name)
+            print "Tag name:" + cell
```
Comment (ajmalmsali) [inline Main.py]: you kept this for debugging ryt?

File: connections.py (hunk 1 above)
Comment (akhilvenugopal) [reply connections.py]: its for deleting selected tags from a group.

**Comment by ajmalmsali** [comment]
@akhilvenugopal  done with the changes?

**Comment by ajmalmsali** [comment]
@akhilvenugopal  remove the opc_client.db file.

**Pull request merged**